package.json

```diff
@@ -34,7 +34,8 @@
     "lint": "run-p lint:*",
     "lint:js": "eslint --report-unused-disable-directives .",
     "lint:exports": "npm run build && npm pack && attw gravity-ui-app-layout-${npm_package_version}.tgz",
-    "lint:other": "npm run prettier -- --check",
+    "lint:prettier": "npm run prettier -- --check",
+    "lint:prettier:fix": "npm run prettier -- --write",
```

Comment:
Yes, when starting `lint`, both will start. I've fixed the scripts a bit, I'll explain my logic. I want `--write `not to be automatic and the developer started it consciously; and like earlier when `lint` starts, it will highlight all errors without making changes.

Reply:
Why does prettier need fixing but eslint doesn't? Do we really need this script? On commit, both prettier and eslint are started with the autofix flag.

Reply:
I used it when I was checking my code, and when errors were found, it helped fix prettier errors. If you think that it is not necessary, I can delete

Reply:
done